website/content/en/highlights/2022-06-10-vrl-vm-runtime-removal.md

```diff
@@ -0,0 +1,62 @@
+---
+date: "2022-06-10"
+title: "Removal of VRL VM beta runtime."
+description: "Default AST runtime got a speed bump."
```

Comment:
`speed bump` - as I read it is usually a construct to make vehicles go slower, maybe we just want to say something like "Default AST runtime performance has improved"